src/diffusion_for_multi_scale_molecular_dynamics/models/axl_diffusion_lightning_model.py

```diff
@@ -417,11 +454,46 @@ def _get_coordinates_target_normalized_score(
         delta_relative_coordinates = map_relative_coordinates_to_unit_cell(
             noisy_relative_coordinates - real_relative_coordinates
         )
-        target_normalized_scores = get_sigma_normalized_score(
+        target_normalized_scores = get_coordinates_sigma_normalized_score(
             delta_relative_coordinates, sigmas, kmax=self.hyper_params.kmax_target_score
         )
         return target_normalized_scores
 
+    def _get_lattice_target_normalized_score(
+        self,
+        noisy_lattice_parameters: torch.Tensor,
+        real_lattice_parameters: torch.Tensor,
+        sigmas_n: torch.Tensor,
+        alpha_bars: torch.Tensor,
+    ) -> torch.Tensor:
+        """Get target normalized score for the lattice parameters.
+
+        It is assumed that the inputs are consistent, ie, the noisy lattice parameters correspond
```

Comment:
This is a copy-paste from the `relative coordinates` method. Rephrase the text to be relevant for the lattice parameters (the input is `sigmas_n`, there is no notion of atoms here, etc).

Reply:
fixed